modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/context/reducer.js

```diff
@@ -11,6 +11,7 @@
 
 export const actionTypes = {
 	CHANGE_STEP: 'CHANGE_STEP',
+	UPDATE_DXPCLOUD_ACTIVATION_STATUS: 'UPDATE_DXPCLOUD_ACTIVATION_STATUS',
```

Comment:
Action name should be an action together with state name
```suggestion
	UPDATE_DXP_CLOUD_ACTIVATION_SUBMITTED_STATUS: 'UPDATE_DXP_CLOUD_ACTIVATION_SUBMITTED_STATUS',
```